devportalservice/devportalservice.go

```diff
@@ -254,3 +255,21 @@ func validateTestDevice(deviceList []TestDevice) (validDevices, duplicatedDevice
 
 	return validDevices, duplicatedDevices
 }
+
+// WritePrivateKeyToFile writes the contents of the private key to a temporary file and returns its path
+func (c *APIKeyConnection) WritePrivateKeyToFile() (string, error) {
+	privatekeyFile, err := os.CreateTemp("", "apiKey*.p8")
```

Comment:
why is the base name `apiKey*.p8` it is usually in `AuthKey_<ISSUER_ID>.p8` format?

Reply:
This way an unique temp file, in case multiple Steps would try to create one.